AlinaFirsova/Homework 1/Homework_1_Firsova.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE HTML>
+<html>
+ <head>
+  <meta charset="utf-8">
+  <title>Cubes</title>
+<style type="text/css">
+ @import"style.css"
+  </style>
+ </head>
+<body background="img\back.png">
+<div class="center">
+
+<!�������� �������>
+  <table cellspacing="0" cellpadding="0" height="408" width="612">
+
+<! ������ ������>
+   <tr align="center"  valign="middle" height="204">
+<td bgcolor="#00c4ef"  width="204"><img src="img\rainy_icon.png"><br><br><front_main>CHICAGO / 23°</front_main></td>

```

Comment:
@AlinaFirsova Please don't use front_main and front_add tags. Thay can be replaced by span or p